assets/xml/audio/samplebanks/SampleBank_0.xml

```diff
@@ -575,45 +575,45 @@
     <Sample Name="SAMPLE_0_574" FileName="Sample574" Offset="0x3BF500" SampleRate="24000" BaseNote="GF3"/>
-    <Sample Name="SAMPLE_0_575" FileName="Sample575" Offset="0x3C0F20" SampleRate="22050" BaseNote="G3"/>
-    <Sample Name="SAMPLE_0_576" FileName="Sample576" Offset="0x3C83F0" SampleRate="22050" BaseNote="DF3"/>
-    <Sample Name="SAMPLE_0_577" FileName="Sample577" Offset="0x3D0630" SampleRate="24000" BaseNote="C4"/>
-    <Sample Name="SAMPLE_0_578" FileName="Sample578" Offset="0x3D42E0" SampleRate="32000" BaseNote="DF3"/>
-    <Sample Name="SAMPLE_0_579" FileName="Sample579" Offset="0x3D9670" SampleRate="22000" BaseNote="C4"/>
-    <Sample Name="SAMPLE_0_580" FileName="Sample580" Offset="0x3DE410" SampleRate="32000" BaseNote="C4"/>
-    <Sample Name="SAMPLE_0_581" FileName="Sample581" Offset="0x3E0BA0" SampleRate="32000" BaseNote="D4"/>
-    <Sample Name="SAMPLE_0_582" FileName="Sample582" Offset="0x3E3330" SampleRate="32000" BaseNote="C4"/>
+    <Sample Name="VocalAahs" FileName="vocal_aahs" Offset="0x3C0F20" SampleRate="22050" BaseNote="G3"/>
+    <Sample Name="ResonantVocalAahs" FileName="resonant_vocal_aahs" Offset="0x3C83F0" SampleRate="22050" BaseNote="DF3"/>
+    <Sample Name="Harp2" FileName="harp2" Offset="0x3D0630" SampleRate="24000" BaseNote="C4"/>
+    <Sample Name="Glockenspiel" FileName="glockenspiel" Offset="0x3D42E0" SampleRate="32000" BaseNote="DF3"/>
+    <Sample Name="BassDrum1" FileName="bass_drum1" Offset="0x3D9670" SampleRate="22000" BaseNote="C4"/>
+    <Sample Name="SnareDrum1" FileName="snare_drum1" Offset="0x3DE410" SampleRate="32000" BaseNote="C4"/>
+    <Sample Name="SnareDrum2" FileName="snare_drum2" Offset="0x3E0BA0" SampleRate="32000" BaseNote="D4"/>
+    <Sample Name="Cymbal" FileName="cymbal" Offset="0x3E3330" SampleRate="32000" BaseNote="C4"/>
     <Sample Name="SAMPLE_0_583" FileName="Sample583" Offset="0x3EBAA0" SampleRate="22050" BaseNote="C4"/>
     <Sample Name="SAMPLE_0_584" FileName="Sample584" Offset="0x3EF2C0" SampleRate="32000" BaseNote="EF4"/>
     <Sample Name="SAMPLE_0_585" FileName="Sample585" Offset="0x3F0780" SampleRate="32000" BaseNote="AF3"/>
     <Sample Name="SAMPLE_0_586" FileName="Sample586" Offset="0x3F21D0" SampleRate="32000" BaseNote="BF1"/>
-    <Sample Name="SAMPLE_0_587" FileName="Sample587" Offset="0x3F3980" SampleRate="32000" BaseNote="C4"/>
-    <Sample Name="SAMPLE_0_588" FileName="Sample588" Offset="0x3F43C0" SampleRate="32000" BaseNote="F3"/>
-    <Sample Name="SAMPLE_0_589" FileName="Sample589" Offset="0x3F9010" SampleRate="22050" BaseNote="A5"/>
-    <Sample Name="SAMPLE_0_590" FileName="Sample590" Offset="0x4009E0" SampleRate="32000" BaseNote="C4"/>
-    <Sample Name="SAMPLE_0_591" FileName="Sample591" Offset="0x40A1A0" SampleRate="32000" BaseNote="EF3"/>
+    <Sample Name="DrumSidestick" FileName="drum_sidestick" Offset="0x3F3980" SampleRate="32000" BaseNote="C4"/>
+    <Sample Name="Trombone" FileName="trombone" Offset="0x3F43C0" SampleRate="32000" BaseNote="F3"/>
```

Comment:
How do you even differentiate this instrument from the one you called trumpet. They sound like the same instrument but playing different notes

Reply:
I didn't actually make these names. I took them from the old Audio XML from OOT. I am fine with changing them.